README.md

```diff
@@ -424,3 +424,25 @@ or accessibility checks must be expected. A new CR instance should be used to re
 Currently, the file retrievals are limited to GitHub repositories only, and files size up to 2 Megabytes.
 
 
+## User OAuth configuration
+
+In situations, when OAuth configurations set for SPI does not fit user's use case (like on-prem installations), one may define it's own OAuth service provider configuration in the Kubernetes secret:
```

Comment:
grammar nazi: "In situations when the OAuth configuration of SPI does not fit the user's use case (like on-prem installations), one may define their own OAuth service provider configuration using a Kubernetes secret:"

Reply:
fixed